File: ambiente/atividades/entregavel1/javascript/somatorio.js
// Somatório: Soma de um conjunto de números.

let numeros = [1, 2, 3, 4, 5]; // altere os valores do array para calcular o somatório de outros números
let soma = 0;
for(let i=0;i<numeros.length;i++){
    soma += numeros[i];
}
console.log("Somatório: " + soma);

File: ambiente/atividades/entregavel1/java/Somatorio.java
public class Somatorio {
    public static void main(String[] args) {
        int[] numeros = {1, 2, 3, 4, 5}; // altere os valores do array para calcular o somatório de outros números
        int soma = 0;
        for(int i=0;i<numeros.length;i++){
            soma += numeros[i];
        }
        System.out.println("Somatório: " + soma);
    }
}
